index.html

```diff
@@ -164,6 +197,14 @@ <h3>
         <!-- DOES IT BREAK THE NATURAL FLOW? WILL THE CONTENT PROVIDE THE HEIGHT? -->
       </main>
       <aside class="resources-sidebar">
```

Comment:
Nice use of the aside & main tag we can quickly identify the different sections of the page